### ISSUE TITLE
[sequence question] How should I run this code if I set the sequence_length to 4？

### ISSUE THREAD

**Issue by bpbdq** [jiwidi/Behavior-Sequence-Transformer-Pytorch#7]
Thanks for sharing this code, and I have a question with it. 

When I run this code on Colab with the original setup，it would run perfectly. But when I change the sequence_length = 4 and step_size = 2 (just like the description in  `With sequence_length of 4 and step_size of 2, we end up with 498,623 sequences.`), the code has an error with 
 `Sizes of tensors must match except in dimension 2. Expected size 4 but got size 8 for tensor number 1 in the list.`
in line ` transfomer_features = torch.cat((transfomer_features, positional_embedding), dim=2)`
What should I do to get it to work properly?

P.S.  Sorry for my poor English.
